modules/bmc/redfish/hpe.rb

```diff
@@ -0,0 +1,13 @@
+module RedfishVendorOverridesHPE
+
+  def poweroff(soft=false)
+    poweraction( soft ? 'PushPowerButton' : 'ForceOff' )
+  end
+
+  # -- See note about dynamic bootdevices in redfish.rb --
+  #def bootdevices
+  #  # older HPs have the list of supported values at a different location
+  #  host.Systems.Members[0].Boot.BootSourceOverrideSupported || super
+  #end
```

Comment:
Do you want to delete this?

Reply:
It's not used currently, but I was leaving it as tested code (or at least I tested it) for the day when boot devices might become more dynamic than the current four hardcoded selections. The other vendors (and recent enough HPs) didn't need anything special to retrieve the list of possible boot devices. I'll delete it if you would prefer that to leaving the comment. Or I can add some more comments describing what I just said.

Reply:
Feel free to leave it, I just thought it's some leftover.